### DISCUSSION TITLE
在项目文件下编译的文件和通过xrepo 安装的包里的文件md5sum不一样

### DISCUSSION THREAD

**Discussion by jkawnkwh85** [xmake-io/xmake#3364] [Q&A] [answered]
编译shared library，确认了在原项目使用的git commit 和xrepo使用的commit一致。
比较了`xmake f -vD`的输出，发现xrepo在编译时默认启用了`cxflags = -fPIC`, 请问这是预期中的吗？

**Comment by waruqi** [comment, marked as answer]
是的
